### ISSUE TITLE
Setting up new autoexports vs managing existing autoexports

### ISSUE THREAD

**Issue by magnusaarnes** [retorquere/zotero-better-bibtex#2301]
I have set my library to "Keep updated" with the "Better BibTeX" format, but when I add a new entry to my library, the exported format is changed to Better BibLaTeX (or similar). This is noticeable as regular websites are exported with the `@online` tag, and not `@misc`. 

The format is still set to Better BibTeX in the Export-settings. When I export it manually again, the format is as it should be.

Debug log ID: 5DPNPJNF-euc

**Comment by retorquere** [comment]
I can't currently replicate the problem locally. A new build with more logging will drop here in 15 minutes or so, can you reproduce the problem and send a new log from that?

**Comment by retorquere** [comment]
I see only one auto-export in your setup, and it's for Better BibLaTeX, not better bibtex.

**Comment by magnusaarnes** [comment]
I see now that in the preferences, my library is exported as Better BibLaTeX. I find this strange as when I click File -> Export Library, the format is set to Better BibTeX.

I deleted the automatic export from the preferences and exported again, and now the preferences say Better BibTeX also. I remember first exporting as Better BibLaTeX by mistake, and this was not changed to Better BibTeX when I changed the format in the Export Library dialog box. I don't know if this is user error from me, but it seems a bit odd that it is that way. 

![image](https://user-images.githubusercontent.com/55587209/199023191-07e75860-0e1f-45d7-b988-778efea6fee2.png)
![image](https://user-images.githubusercontent.com/55587209/199023254-846c0ae0-6b5e-4538-8aed-703557a88bf3.png)
